packages/autocomplete-js/src/autocomplete.ts

```diff
@@ -55,28 +55,6 @@ export function autocomplete<TItem extends BaseItem>({
     classNames,
   });
 
-  // This batches state changes to limit DOM mutations.
-  // Every time we call a setter in `autocomplete-core` (e.g., in `onInput`),
-  // the core `onStateChange` function is called.
-  // We don't need to be notified of all these state changes to render.
-  // As an example:
-  //  - without debouncing: "iphone case" query → 85 renders
-  //  - with debouncing: "iphone case" query → 12 renders
-  const debouncedOnStateChange = debounce((state: AutocompleteState<TItem>) => {
```

Comment:
this removal seems unrelated to this PR, wrong rebase?

Reply:
It doesn't appear in the PR changes, did you comment the merge commit?